samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,377 @@
+    Connected isochronous streams can either transfer data from central to peripheral, peripheral to central, or in both directions.
+
+SDU interval and size
+    The SDU interval specifies how often a service data unit (SDU) is transferred.
+    The SDU interval and size as well as the number of streams are limited by the bandwidth of the link layer.
+    In this sample the SDU interval is configured with the Kconfig option :kconfig:option:`CONFIG_SDU_INTERVAL_US`.
+
+Maximum transport latency and retransmission count
+    Upon setting up an isochronous stream as a central or a broadcaster, the application provides a maximum transport latency and retransmission count.
+    The Bluetooth controller will use the provided parameters to select link layer parameters that satisfy these constraints.
+    This involves selecting an isochronous interval, PDU sizes, and isochronous parameters such as burst number (BN), number of subevents (NSE), and flush timeout (FT) or pretransmission offset (PTO).
+    For more details about how the SoftDevice Controller selects these parameters, see :ref:`Parameter selection <iso_parameter_selection>`.
+
+Presentation delay
+    The receiver(s) of isochronous data may receive their data at different points in time, but need to render/present it synchronously.
+    Presentation delay represents the amount of time that is added to the received timestamp to obtain the presentation time of the data.
+    In this sample the presentation delay is configured with the Kconfig option :kconfig:option:`CONFIG_TIMED_LED_PRESENTATION_DELAY_US`.
+
+For more details about this feature, see `Bluetooth Core Specification Version 5.2 Feature Overview <https://www.bluetooth.com/wp-content/uploads/2020/01/Bluetooth_5.2_Feature_Overview.pdf>`_.
+
+This sample demonstrates the following:
+
+* Any of the possible isochronous topologies with unidirectional data transfer.
+  The topology is selected when the application starts.
+* The transmitter of the data sends it over all configured and connected streams.
+  The number of streams is configured with the :kconfig:option:`CONFIG_BT_ISO_MAX_CHAN` Kconfig option.
+* The receiver of the isochronous data receives data from one transmitter only.
+* The maximum transport latency and retransmission count is configured when the application starts.
+
+
+Sample structure
+================
+
+The sample code is divided into multiple source files which makes it easier to compile in or out certain functionality.
+
+``main.c``
+  The main entry point of the sample allows the user to selects the role and parameters to be used.
+
+``bis_transmitter.c``
+  This file implements a device that sets up a BIS transmitter with a configured number of streams.
+  That is:
+
+  1. Configure and start an extended advertiser.
+     The extended advertiser advertiser data contains the device name.
+  2. Configure and start a periodic advertiser.
+  3. Configures and starts a broadcast isochronous group (BIG).
```

Comment:
```suggestion
  *  Configure and start an extended advertiser.
     The extended advertiser advertiser data contains the device name.
  * Configure and start a periodic advertiser.
  * Configure and start a broadcast isochronous group (BIG).
```
If these are steps that need to happen in this order, use 1. for the first item and then #. for the others.